### PR TITLE
[IMP] crm: make crm.lead phone field editable

### PR DESCRIPTION
**Pull request by mcm-odoo**
Before this commit, the user could not edit the lead phone field
when a customer was assigned. The phone field was related to the
phone of that customer.

Now, the field related to the phone of the customer is removed.
We can edit the phone field of the lead without modifying the customer
one.

task-2083470

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/crm/models/crm_lead.py [hunk 1]
```diff
@@ -266,9 +263,6 @@ def _onchange_partner_id_values(self, partner_id):
                 'city': partner.city,
                 'state_id': partner.state_id.id,
                 'country_id': partner.country_id.id,
-                'email_from': partner.email,
-                'phone': partner.phone,
-                'mobile': partner.mobile,
```
Comment (awa-odoo) [inline addons/crm/models/crm_lead.py]: Shouldn't we keep those though?

Comment (mcm-odoo) [reply addons/crm/models/crm_lead.py]: I re-tested and it didn't happen again

File: addons/crm/models/crm_lead.py [hunk 2]
```diff
@@ -120,9 +120,6 @@ def _default_stage_id(self):
     expected_revenue = fields.Monetary('Prorated Revenue', currency_field='company_currency', store=True, compute="_compute_expected_revenue")
     date_deadline = fields.Date('Expected Closing', help="Estimate of the date on which the opportunity will be won.")
     color = fields.Integer('Color Index', default=0)
-    partner_address_name = fields.Char('Partner Contact Name', related='partner_id.name', readonly=True)
-    partner_address_email = fields.Char('Partner Contact Email', related='partner_id.email', readonly=True)
-    partner_address_phone = fields.Char('Partner Contact Phone', related='partner_id.phone', readonly=True)
     partner_is_blacklisted = fields.Boolean('Partner is blacklisted', related='partner_id.is_blacklisted', readonly=True)
```
Comment (awa-odoo) [inline addons/crm/models/crm_lead.py]: You should be able to remove this one as well.

File: addons/crm/views/crm_lead_views.xml [hunk 3]
```diff
@@ -173,20 +173,13 @@
...
                                         aria-label="Blacklisted" attrs="{'invisible': ['|', ('is_blacklisted', '=', False), ('partner_address_email', '!=', False)]}" groups="base.group_user"/>
                                     <field name="email_from"
-                                        attrs="{'invisible': [('partner_address_email', '!=', False)]}" string="Email" widget="email"/>
+                                        attrs="{'invisible': [('partner_id', '!=', False)]}" string="Email" widget="email"/>
                                     <i class="fa fa-ban" style="color: red;" role="img" title="This email is blacklisted for mass mailing"
                                         aria-label="Blacklisted" attrs="{'invisible': ['|', ('partner_is_blacklisted', '=', False), ('partner_address_email', '=', False)]}" groups="base.group_user"/>
                                     <field name="partner_address_email"
-                                        attrs="{'invisible': [('partner_address_email', '==', False)]}" widget="email" string="Email"/>
+                                        attrs="{'invisible': [('partner_id', '==', False)]}" widget="email" string="Email"/>
```
Comment (tde-banana-odoo) [inline addons/crm/views/crm_lead_views.xml]: '==' ? Does that work ? Should probably be '=' like all other domains :) .

Comment (mcm-odoo) [reply addons/crm/views/crm_lead_views.xml]: It seemed to work but I've fixed it.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**